dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/AnnotationBean.java

```diff
@@ -96,6 +96,14 @@ public void postProcessBeanFactory(ConfigurableListableBeanFactory beanFactory)
                 // spring 2.0
             }
         }
+
+
+        //
+        if(!beanFactory.containsBean(DynamicReferenceService.class.getName())) {
+            //注册服务bean
+            beanFactory.registerSingleton(DynamicReferenceService.class.getName(), new DynamicReferenceService(this));

```

Comment:
What I am worried about is that the DynamicReferenceService injection is overrided， eg: I have two reference bean, 
``` java
     Consumer A (ReferenceBean #A): 
    // inject service
    @Autowired
    private DynamicReferenceService dynamicReferenceService;

     Consumer B (ReferenceBean #B): 
    // inject service
    @Autowired
    private DynamicReferenceService dynamicReferenceService;
```
Spring container contains "new DynamicReferenceService(ReferenceBean #B)" only, previous new DynamicReferenceService(ReferenceBean #A) instance maybe lost, when you invoke any service method in Consumer A (ReferenceBean #A), you hold DynamicReferenceService(ReferenceBean #B) instance actually, maybe trigger some bugs?